pkg/projects/Microsoft.NETCore/Microsoft.NETCore.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.0.3</Version>
+    <Version>5.0.2</Version>
```

Comment:
We are shipping 5.0.2 in the 1.0.0 branch as well. Our previous UWP MP 5.3.1 contained a reference to Microsoft.NETCore 5.0.2 which is why I made this change. It seems like we don't have much choice except to keep this at 5.0.2 because 5.0.3 is still pre-release.